src/chromium/crBrowser.ts

```diff
@@ -135,27 +125,47 @@ export class CRBrowser extends platform.EventEmitter implements Browser {
           context.emit(Events.CRBrowserContext.BackgroundPage, event);
           break;
         }
-        case 'service_worker': {
-          const serviceWorker = await target.serviceWorker();
-          context.emit(Events.CRBrowserContext.ServiceWorker, serviceWorker);
-          break;
-        }
       }
     } catch (e) {
```

Comment:
I think we can remove this catch now, since we use pageOrError().

Reply:
Done.